src/wallet/rpc/spend.cpp

```diff
@@ -333,7 +333,7 @@ RPCHelpMan sendmany()
                             {"address", RPCArg::Type::STR, RPCArg::Optional::OMITTED, "Subtract fee from this address"},
                         },
                     },
-                    {"replaceable", RPCArg::Type::BOOL, RPCArg::DefaultHint{"wallet default"}, "Allow this transaction to be replaced by a transaction with higher fees via BIP 125"},
+                    {"replaceable", RPCArg::Type::BOOL, RPCArg::DefaultHint{"wallet default"}, "Signal that this transaction can replaced by a transaction (BIP 125)"},
```

Comment:
I wonder if you should update the argument descriptions similarly  in  `TransactionDescriptionString()` (L420 in `src/wallet/rpc/transactions.cpp`) and in `MempoolEntryDescription()` (L258 in `src/rpc/mempool.cpp`)

If yes, can be done as a follow-up.

Reply:
Thanks, done in #25902